Dockerfile

```diff
@@ -0,0 +1,68 @@
+# syntax=docker/dockerfile:1
+ARG PYTHON_VERSION=3.10
+
+##################################################
+#                   Python                       #
+##################################################
+FROM python:${PYTHON_VERSION}-slim-buster AS signals_python
+
+ENV PYTHONUNBUFFERED 1
+ENV DJANGO_SETTINGS_MODULE=signals.settings
+ARG DJANGO_SECRET_KEY=insecure_docker_build_key
+
+WORKDIR /app/app
+
+RUN useradd --no-create-home signals
+
+COPY api/requirements.txt /app/requirements.txt
+
+RUN set -eux;  \
```

Comment:
I see for testing we are using a slightly different package list: https://github.com/Amsterdam/signals/blob/5396c95daf9249748768aded53bcab885cea7265/.github/workflows/ci.yml#L30

Reply:
You're right I copied this from the original base image, we probably don't need all of those, I'll see what we can remove here.

Reply:
I removed the above packages. The test environment also runs on Ubuntu, so I'm not sure it would leave us in a comparable state given that we use Debian in our images. We can always look at removing even more packages from the dependency list in the future. Or maybe even switch to using Alpine based images. :smiley: